tests/integration/test_kyber.py

```diff
@@ -0,0 +1,139 @@
+import sys
+import time
+from pathlib import Path
+
+import psutil
+import pytest
+import requests
+from eth_utils import to_bytes
+from tests.constants import TESTING_TEMP_FOLDER
+from tests.integration import kyber_snapshot_addresses
+
+from raiden_installer.account import Account
+from raiden_installer.ethereum_rpc import make_web3_provider
+from raiden_installer.kyber.web3.constants import NETWORK_ADDRESS_MODULES_BY_CHAIN_ID
+from raiden_installer.network import NETWORK_CLASSES, Network
+from raiden_installer.token_exchange import ExchangeError, Kyber
+from raiden_installer.tokens import Erc20Token, TokenAmount
+from raiden_installer.transactions import get_token_balance
+
+FAKE_BLOCKCHAIN_PATH = Path("tests", "fake_blockchain")
+PORT = "8546"
+CHAIN_ID = 1337
+# Args as specified by the Kyber snapshot documentation
+GANACHE_COMMAND = [
+    "node",
+    "node_modules/ganache-cli/cli.js",
+    "--db",
+    "node_modules/kyber-network-workshop/db",
+    "--accounts",
+    "10",
+    "--defaultBalanceEther",
+    "1000",
+    "--mnemonic",
+    "gesture rather obey video awake genuine patient base soon parrot upset lounge",
+    "--networkId",
+    "5777",
+    "--port",
+    PORT
+]
+
+TESTING_KEYSTORE_FOLDER = TESTING_TEMP_FOLDER.joinpath("keystore")
+# Prefilled account from Kyber Ganache snapshot
+WALLET_PRIVATE_KEY = 0x979d8b20000da5832fc99c547393fdfa5eef980c77bfb1decb17c59738d99471
+
+KNC_TOKEN = Erc20Token(
+    ticker="KNC",
+    wei_ticker="KEI",
+    addresses={"ganache": kyber_snapshot_addresses.TokenAddress.KNC.value},
+    network="ganache"
+)
+
+
+def take_snapshot():
+    body = dict(jsonrpc="2.0", method="evm_snapshot", params=[], id=1)
+    response = requests.post(f"http://localhost:{PORT}", json=body)
+    return response.json()["result"]
+
+
+def revert_to_snapshot(snapshot_id):
+    body = dict(jsonrpc="2.0", method="evm_revert", params=[snapshot_id], id=1)
+    requests.post(f"http://localhost:{PORT}", json=body)
+
+
+@pytest.fixture
+def test_account(monkeypatch):
+    monkeypatch.setattr(Account, "generate_private_key", lambda: to_bytes(WALLET_PRIVATE_KEY))
+    account = Account.create(TESTING_KEYSTORE_FOLDER, "test_password")
+    yield account
+    account.keystore_file_path.unlink()
```

Comment:
I'd suggest a contant/fixture for the password, i.e.

```suggestion
@pytest.fixture
def test_password():
    return "test_password"

@pytest.fixture
def test_account(monkeypatch, test_password):
    monkeypatch.setattr(Account, "generate_private_key", lambda: to_bytes(WALLET_PRIVATE_KEY))
    account = Account.create(TESTING_KEYSTORE_FOLDER, test_password)
    yield account
    account.keystore_file_path.unlink()
```
```suggestion
    account = Account.create(TESTING_KEYSTORE_FOLDER, "test_password")
```

Reply:
Nice!